tortillas/log_analyzer.py

```diff
@@ -142,8 +142,11 @@ def analyze(
                 else TestStatus[config_entry.set_status]
             )
 
-            if not logs:
-                continue
+            # fixme: this is flawed, why is this here?
+            #  when tortillas gets no logs and and aborts, it can't detect incorrect or missing exit codes
+            #  as a result, every test with incorrect exit codes passes
+            # if not logs:
+            #     continue
```

Comment:
I think it cashed a some point because logs where empty. I was lazy and just added this.

Nitpick:
I think we should add a check before `result.add_errors(logs[0:1], status)` just so logs has at least one line. All the other cases seem to handle logs being empty.

Reply:
nvm... `[][0:1] == []` passt also eh.